project/src/main/webapp/about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
```

Comment:
As discussed f2f, I think we'll have to add the navigation from the other pages to the top of this one and add link to about.html to the other pages as well.

Reply:
You could also write this information in the Home page instead.

Reply:
This is a good idea. Michael and I are working on this right now for the demo.